resin/requests.go

```diff
@@ -98,7 +98,11 @@ func Signup(endpoint, email, password string) (token string, err error) {
 func GetApps(endpoint, token string) (apps []map[string]interface{}, err error) {
 	client := pinejs.NewClientWithToken(endpoint+"/v1", token)
 	apps = []map[string]interface{}{map[string]interface{}{"pinejs": "application"}}
-	err = client.List(&apps)
+	t, e := util.ScanDeviceTypeSlug("/etc/os-release")
+	if e != nil {
+		return "", fmt.Errorf("Could not get device type: %s", e)
+	}
+	err = client.List(&apps, pinejs.NewQueryOptions(pinejs.Filter, "device_type eq '"+t+"'")...)
```

Comment:
This is hardcoding this directory, we have `OSRELEASE_PATH` for this. And no need for even further reducing var names until they're completely unreadable, plus more readable to use `fmt.Sprintf()` rather than concatenation, e.g.:

```go
var deviceType string
if deviceType, err = util.ScanDeviceTypeSlug(OSRELEASE_PATH); err != nil {
        return "", fmt.Errorf("Could not get device type in GetApps: %s", err)
}

deviceTypeQuery := fmt.Sprintf("device_type eq '%s'", deviceType)
opts := pinejs.NewQueryOptions(pinejs.Filter, deviceTypeQuery)
err = client.List(&apps, opts...)
```